components/02-elements/form/select/_select.scss

```diff
@@ -1,17 +1,23 @@
-$select-border-color: #d9d9d9 !default;
-$select-padding: 10px 15px !default;
-$select-border-radius: $border-radius !default;
+$select-margin              : 10px !default;
+$select-padding             : 10px 15px !default;
+$select-border-color        : #d9d9d9 !default;
+$select-border-radius       : $border-radius !default;
+$select-list-single-border  : 1px solid $select-border-color !default;
+$select-list-dropdown-border: 1px solid $select-border-color !default;
```

Comment:
if border it's the same, let's do one variable for border

Reply:
changed and checked in each file